### PR TITLE
Delete `/containerengine/` folder

### PR DESCRIPTION
**Pull request by NimJay**
* Google Kubernetes Engine used to be called "Google Container Engine".
    * See [this Google-internal change](https://critique.corp.google.com/cl/175203021/depot/google3/googledata/devsite/site-cloud/en/_shared/_product_names.html) where the product is renamed.
* Today, all the samples for Google Kubernetes Engine's cloud.google.com tutorials live inside https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/.
* In this pull-request, I am deleting the `/containerengine/` folder and the related `.github/workflows/containerengine-hello-world.yaml` file.

### CODE REVIEW

**Comment by kweinmeister** [comment]
@NimJay do we just need to delete this folder? Any doc references or anything else to worry about?

File: containerengine/hello-world/server.js [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-const http = require('http');
-const handleRequest = function (req, res) {
-  res.writeHead(200);
-  res.end('Hello Kubernetes!');
-};
-const www = http.createServer(handleRequest);
-www.listen(parseInt(process.env.PORT) || 8080);
-
-// [END all]
-module.exports = www;
```
Comment (NimJay) [inline containerengine/hello-world/server.js:27]: This is the only real "sample code" file in this `containerengine` folder, and it doesn't seem to be used anywhere.
I have checked [Google's internal Code Search](https://source.corp.google.com/search?q=containerengine%20hello-world&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental).

File: containerengine/hello-world/README.md [hunk 2]
```diff
@@ -1,29 +0,0 @@
-<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
-
-# Google Container Engine World sample
-
-This is a basic hello world sample.
-
-View the [documentation][docs] or the [source code][code].
-
-[docs]: https://cloud.google.com/container-engine/docs/quickstart
```
Comment (NimJay) [inline containerengine/hello-world/README.md:9]: This [quickstart](https://cloud.google.com/container-engine/docs/quickstart) uses `us-docker.pkg.dev/google-samples/containers/gke/hello-app:1.0` — which is hosted and maintained at [github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/main/hello-app](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/main/hello-app).

**Comment by NimJay** [comment]
Hi @kweinmeister,

Thanks for looking into this pull-request!
I just added additional info/comments, and resolved the conflicts.
It's now ready for review.

> "Any doc references or anything else to worry about?"

I am confident this folder can be deleted. It only contains 1 meaningful "sample code" file — and it doesn't seem used.

I checked Code Search and the [quickstart](https://cloud.google.com/kubernetes-engine/docs/deploy-app-cluster#create_the_deployment) that the `README.md`s (being deleted by this pull-request) are referring to. The quickstart does not use anything from this [`GoogleCloudPlatform/nodejs-docs-samples`](https://github.com/GoogleCloudPlatform/nodejs-docs-samples) repo.

**Review by kweinmeister — APPROVED** [review]

**Comment by kweinmeister** [comment]
Thanks for cleaning this up, @NimJay!

**Pull request merged**